### PR TITLE
Small updates to boilerplate docs

### PR DESCRIPTION
**Pull request by MarciaRieferJohnston**
Acrolinx says to avoid using the word "action" when referring to an AWS operation. Vinod confirms that it makes sense to remove the word here.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: planning_deployment.adoc [hunk 1]
```diff
@@ -90,7 +90,7 @@ endif::template_ec2[]
 
 ==== IAM permissions
 //todo: scope of least-privilege
```
Comment (dowen12) [inline planning_deployment.adoc:92]: This needs to be removed from here and should not be in the boilerplate files.

**Review by dowen12 — DISMISSED** [review]

File: planning_deployment.adoc (hunk 1 above)
Comment (vsnyc) [reply planning_deployment.adoc:92]: I think we can resolve this comment for this PR and look only at the change proposed. @andrew-glenn may have left that as a placeholder for future work to automatically generate minimum privileges required to launch a quick start.

Comment (andrew-glenn) [reply planning_deployment.adoc:92]: This was a placeholder I put in there. It's out of scope for this PR.

Comment (vsnyc) [reply planning_deployment.adoc:92]: Agree

**Review by vsnyc — DISMISSED** [review]
Looks good to me.

**Review by troy-ameigh — APPROVED** [review]

**Review by davmayd — APPROVED** [review]

**Review by censullo — APPROVED** [review]

**Review by DelfinGala — APPROVED** [review]

**Review by andrew-glenn — CHANGES_REQUESTED** [review]
"post deployment" / "post-deployment" is an industry accepted term. Technical context is lost when it's condensed.

**Review by andrew-glenn — APPROVED** [review]

**Review by dowen12 — APPROVED** [review]

**Pull request merged**